cypress/integration/learn/responsive-web-design/claim-cert-from-learn.js

```diff
@@ -0,0 +1,99 @@
+/* global cy */
+
+const projects = {
+  superBlock: 'responsive-web-design',
+  block: 'responsive-web-design-projects',
+  challenges: [
+    {
+      slug: 'build-a-tribute-page',
+      solution: 'https://codepen.io/moT01/pen/ZpJpKp'
+    },
+    {
+      slug: 'build-a-survey-form',
+      solution: 'https://codepen.io/moT01/pen/LrrjGz?editors=1010'
+    },
+    {
+      slug: 'build-a-product-landing-page',
+      solution: 'https://codepen.io/moT01/full/qKyKYL/'
+    },
+    {
+      slug: 'build-a-technical-documentation-page',
+      solution: 'https://codepen.io/moT01/full/JBvzNL/'
+    },
+    {
+      slug: 'build-a-personal-portfolio-webpage',
+      solution: 'https://codepen.io/moT01/pen/vgOaoJ'
+    }
+  ]
+};
+
+describe('Responsive Web Design Superblock', () => {
+  before(() => {
+    cy.exec('npm run seed');
+    cy.login();
+    cy.visit('/learn/responsive-web-design');
+  });
+  describe('Before submitting projects', () => {
+    it('should have a card with href "claim-cert-block"', () => {
+      cy.get('a[href="#claim-cert-block"]').scrollIntoView();
+      cy.get('a[href="#claim-cert-block"]').should('be.visible');
+    });
+
+    it('should have an anchor element with the text "Claim Certification", and class "disabled"', () => {
+      cy.get('a.disabled').should('be.visible');
+      cy.get('a.disabled').should('have.text', 'Claim Certification');
+    });
+
+    it('should have an unordered list with class "map-challenges-ul" containing 5 items', () => {
+      cy.get('a[href="#claim-cert-block"]')
+        .parent()
+        .parent()
+        .find('ul.map-challenges-ul')
+        .should('be.visible');
+      cy.get('a[href="#claim-cert-block"]')
+        .parent()
+        .parent()
+        .find('ul.map-challenges-ul')
+        .children()
+        .should('have.length', 5);
```

Comment:
This is a bit low-level.  Can this be written without needing to know the precise structure of the dom?  How about using a data-* attribute to find the relevant bits?

Reply:
I will spend more time looking into it 👍

Reply:
Let me know if you'd like to pair/justify a bit further.